src/icalendar/cal.py

```diff
@@ -333,7 +333,7 @@ def from_ical(cls, st, multiple=False):
                 component = stack[-1] if stack else None
                 if not component or not component.ignore_exceptions:
                     raise
-                component.errors.append((None, str(e)))
+                component.errors.append((None, repr(e)))
```

Comment:
Right, sorry, here I go suggesting Python 3 compatibility with stuff getting deprecated since Python 2.6.
The obvious solution then is to use six:
```python
import six

if six.PY3:
    unicode = str
...
component.errors.append((None, unicode(e)))
```